### PR TITLE
feat(ResponsiveLayout): allow background color

### PR DESCRIPTION
**Pull request by atabel**
https://github.com/Telefonica/mistica-design/discussions/885

### CODE REVIEW

File: src/responsive-layout.tsx [hunk 1]
```diff
@@ -2,24 +2,40 @@ import * as React from 'react';
...
+}) => {
+    const isInverseOutside = useIsInverseVariant();
     return (
-        <div
-            className={classnames(styles.container, className)}
-            {...getPrefixedDataAttributes(dataAttributes)}
-        >
-            <div className={fullWidth ? styles.fullWidth : styles.responsiveLayout}>{children}</div>
-        </div>
+        <ThemeVariant isInverse={isInverse || isInverseOutside}>
```
Comment (sergivillar) [inline src/responsive-layout.tsx:28]: if `isInverseOutside` is already true, is this ThemeVariant needed?

**Review by sergivillar — APPROVED** [review]

**Review by yceballost — APPROVED** [review]
✨

File: src/responsive-layout.tsx (hunk 1 above)
Comment (atabel) [reply src/responsive-layout.tsx:28]: not really, but it's simpler this way that rendering the `ThemeVariant` conditionally

**Review by pladaria — APPROVED** [review]

**Pull request merged**